### PR TITLE
Interacting with daemon through socket

### PR DESCRIPTION
**Pull request by Camerooooon**

### CODE REVIEW

**Comment by Camerooooon** [comment]
This is a big patch, @JakeRoggenbuck can you give a review before merge?

Stuff to test:

`acs daemon status`
`acs daemon enable`
`acs daemon disable`
`acs daemon toggle`

Do all the commands function actually and adequately enable/disable the daemon?

**Comment by JakeRoggenbuck** [comment]
> 

Yay! Thanks so much. Testing now

**Comment by JakeRoggenbuck** [comment]
p.s. thank you so much for the great error messages in earlier patches. 

image

Very good error.

**Comment by JakeRoggenbuck** [comment]
image

image

File: src/args.rs [hunk 1]
```diff
@@ -222,6 +244,95 @@ pub fn parse_args(config: config::Config) {
     };
 
     match ACSCommand::from_args() {
+        ACSCommand::Daemon { control } => match control {
```
Comment (JakeRoggenbuck) [inline src/args.rs:247]: We might want to put this into interface in future, but that is out of scope.

**Review by JakeRoggenbuck — APPROVED** [review]
Yay!! This is super exciting!

**Comment by JakeRoggenbuck** [comment]
Awesome seeing this feature being built so well.

**Comment by Camerooooon** [comment]
Thank you Jake.

**Pull request merged**